### DISCUSSION TITLE
Bootstrap without LESS

### DISCUSSION THREAD

**Discussion by Im-PJ** [Dotnet-Boxed/Templates#168] [Q&A]
I have no experience with LESS. Infact not much with CSS pre-processors. I want to use this template with bootstrap and vanilla css. Is it possible and how to do it? Sorry its not issue just a question

**Comment by RehanSaeed** [comment]
Not possible. No plans to implement it either. Plain CSS is not a workable solution for anything but super simple projects. I recommend learning LESS or SASS, it's worth the 15 mins.
